src/MAUI/Maui.Samples/AppShell.xaml

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<Shell
+    x:Class="ArcGIS.AppShell"
+    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    Shell.TabBarIsVisible="False">
+    <Shell.FlyoutHeader>
+        <Label
+            Margin="0,5"
+            FontAttributes="Bold"
+            HorizontalTextAlignment="Center"
+            Text="Categories"
```

Comment:
This label isn't visible in MacCatalyst oddly.

Reply:
Actually, this was just because my MacCatalyst window wasn't in fullscreen mode.